src/components/Form/Form.styles.js

```diff
@@ -28,11 +28,8 @@ export const Button = styled.button`
   &:disabled {
     background-color: ${(props) => props.theme.color.indigo200};
     box-shadow: ${(props) => props.theme.boxShadow.none};
-
-    &:hover {
-      background-color: ${(props) => props.theme.color.indigo200};
-    }
   }
+
   ${(props) => props.styles};
```

Comment:
What is this for?

Reply:
I didn't question it, it seems to be what you are describing though.

Reply:
Yes, that is exactly what that prop is for